src/components/Switch/Switch.tsx

```diff
@@ -19,6 +20,12 @@ export type SwitchProps = NativeAttributes<'input'> & {
   uncheckedText?: string;
 };
 
+const switchFocusStyles = {
+  '& .pill': {
```

Comment:
There's probably a more CSS-in-JS way to select the nested pill than a CSS class, not sure. I've always found this to be tricky and confusing in Emotion / Styled Components / etc, even though the CSS itself is simple

Reply:
Why don't we simply put the `_focusWithin` on the `Flex` component itself? Tested it and it works just fine + we avoid the need for a CSS selector?